src/metabase/driver.clj

```diff
@@ -462,6 +466,7 @@
 
 (defmethod supports? [::driver :basic-aggregations] [_ _] true)
 (defmethod supports? [::driver :case-sensitivity-string-filter-options] [_ _] true)
+(defmethod supports? [::driver :date-functions] [_ _] true)
```

Comment:
I'm a little worried that `:date-functions` will be a bit vague for our purposes. We're adding a lot of date functions and we won't be able to make all databases support them. (things like `datediff` and timezone stuff) I wonder if we'll regret the lack of resolution that this imposes on us.

Reply:
so `get-xyz` and `date-add, date-subtract` can work on all DBs so I think this is fine. but for `convertTimezone` and `dateDiff` it's might be a different story.

What do you say I rename this to `date-extract` and use this specifically for `get-xyz` functions?

Reply:
that sounds like a good option. Let's go with that.